### PR TITLE
Add duplicated-imports linter rule and fix it

### PR DESCRIPTION
**Pull request by pgimalac**
### What does this PR do?

Explicitly add the default config of `revive` in `.golangci.yml`, enable `duplicated-imports` rule, and fix it.

### Motivation

Having multiple times the same import is usually a mistake caused by a merge, I've seen it several times while reviewing PRs.
Having the `revive` config explicitly in `.golangci.yml` allows to change it however we want, while preventing disabling it by accident.

### Additional Notes

Having the revive config explicitly means that new lints won't appear just by updating `golangci-lint`, but also that new default rules won't be enabled automatically.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by davidor — APPROVED** [review]
👍 for container-integrations files

**Review by ogaca-dd — APPROVED** [review]

**Review by ahmed-mez — APPROVED** [review]

**Review by kangyili — APPROVED** [review]

File: .golangci.yml [hunk 1]
```diff
@@ -126,3 +126,37 @@ linters-settings:
...
+      - name: redefines-builtin-id
+      - name: superfluous-else
+      - name: time-naming
+      - name: unexported-return
+      - name: unreachable-code
+      - name: unused-parameter
+      - name: var-declaration
+      - name: var-naming
+      # non-default rules:
+      - name: duplicated-imports
```
Comment (amenasria) [inline .golangci.yml:162]: This config is the one from `revive` master right ? Maybe we should use the one from the version of `revive` our current version of `golangci-lint` is using instead ?

**Review by amenasria — APPROVED** [review]
**👏 praise:** ‏Nice PR !!!

LGTM ! We maybe should automate the `revive` -> `golangci-lint` config with an invoke task !

File: .golangci.yml (hunk 1 above)
Comment (pgimalac) [reply .golangci.yml:162]: Nice catch !
I checked, and the only difference between the config from master and the one from 1.3.2 (the version we are using) is that they dropped the `if-return` rule ([PR](https://github.com/mgechev/revive/pull/843)) which shouldn't have been part of the default (it was accidentally added back).
I think we should keep it out of our rules.

**Review by daniel-taf — APPROVED** [review]

**Review by carlosroman — APPROVED** [review]

**Pull request merged**